bot/cogs/reminders.py

```diff
@@ -182,7 +191,9 @@ def __init__(self, bot: Bot):
         # Confirm to the user that it worked.
         await self._send_confirmation(
             ctx,
-            on_success=f"Your reminder will arrive in {humanize_delta(relativedelta(expiration, now))}!"
+            on_success=f"Your reminder will arrive in {humanize_delta(relativedelta(expiration, now))}!",
```

Comment:
Would be more readable IMO if the humanised delta was created on a separate line and assigned to a variable.